lib/android_build/app/src/androidTest/java/com/microsoft/applications/events/maesdktest/LogManagerDDVUnitTest.java

```diff
@@ -248,6 +247,70 @@ public void startDDVonLogManager() {
     LogManager.flushAndTeardown();
   }
 
+  /*
+  Disabling this test since it requires private modules.
+
+  @Test
+  public void startPrivacyGuardWithMultipleLogManagers() {
+    System.loadLibrary("maesdk");
+    Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
+    if (s_client == null) {
+      s_client = new MockHttpClient(appContext);
+    }
+    synchronized (s_client.urlMap) {
+      s_client.urlMap.clear();
+    }
+    OfflineRoom.connectContext(appContext);
+
+    final String token =
+            "0123456789abcdef0123456789abcdef-01234567-0123-0123-0123-0123456789ab-0123";
+    final String contosoToken =
+            "0123456789abcdef9123456789abcdef-01234567-0123-0123-0123-0123456789ab-0124";
+    final String contosoUrl = "https://bozo.contoso.com/";
+    final String contosoName = "ContosoFactory";
+    final String contosoDatabase = "ContosoSequel";
+
+    final ILogger initialLogger = LogManager.initialize(token);
+
+    PrivacyGuardInitConfig config = new PrivacyGuardInitConfig();
+    config.loggerInstance = initialLogger;
+    config.ScanForURLs = false;
+    config.UseEventFieldPrefix = true;
+    // Init Privacy Guard
+    PrivacyGuard.initialize(config);
```

Comment:
@larvacea any thoughts on how to add a check for private module presence? Otherwise, this test fails in the main repo Android-Mac build.